docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -204,9 +204,9 @@ Use the same configuration on all brokers of this cluster.
 
 :::
 
-| Field | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Example Value |
-| ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| store | Set the backup store type. Supported values are [NONE, S3, GCS]. Default value is NONE. When NONE, no backup store is configured and no backup will be taken. Use S3 to use any [S3 compatible storage](https://docs.aws.amazon.com/AmazonS3/latest/API/Type_API_Reference.html). Use GCS to use [Google Cloud Storage](https://cloud.google.com/storage/). This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_BACKUP_STORE`. | NONE          |
+| Field | Description                                                                                                                                                                                                                                                                                                                                                                                                                     | Example Value |
+| ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| store | Set the backup store type. Supported values are [NONE, S3, GCS]. Default value is NONE. When NONE, no backup store is configured and no backup will be taken. Use S3 to use any S3 compatible storage, including, but not limited to, Amazon S3. Use GCS to use [Google Cloud Storage](https://cloud.google.com/storage/). This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_BACKUP_STORE`. | NONE          |
```

Comment:
:bulb: At first I wanted to suggest calling it AWS S3 because this is an AWS product but it looks like their marketing agrees with you and calls it Amazon S3.

Reply:
I have friends at Amazon that also have to look up whether it is Amazon or AWS for each product 😅